dev/com.ibm.jbatch.container/src/com/ibm/jbatch/container/persistence/jpa/JobInstanceEntity.java

```diff
@@ -41,6 +41,7 @@
 import com.ibm.jbatch.container.ws.WSJobInstance;
 
 @NamedQueries({
+                @NamedQuery(name = JobInstanceEntity.GET_JOBINSTANCEIDS_BY_NAME_AND_STATUSES_QUERY, query = "SELECT i.instanceId FROM JobInstanceEntity i WHERE i.batchStatus IN :status AND i.jobName = :name"),
                 @NamedQuery(name = JobInstanceEntity.GET_JOBINSTANCES_SORT_BY_CREATETIME_FIND_ALL_QUERY, query = "SELECT i FROM JobInstanceEntity i ORDER BY i.createTime DESC"),
```

Comment:
So we switched from querying JobExecution batchStatus to  JobInstance batchStatus.   I think that's OK.. I think the one is just a redudantly-added extra column to avoid querying in certain cases.  Right?

Reply:
That's correct